lib/msal-node/docs/managed-identity.md

```diff
@@ -0,0 +1,139 @@
+# Managed Identity with MSALJS
+
+### Note
+
+> This feature is not yet officially supported in MSALJS. An [alpha package](https://www.npmjs.com/package/@azure/msal-node/v/2.3.0-alpha.0) has been upload to npm.
+
+A common challenge for developers is the management of secrets, credentials, certificates, and keys used to secure communication between services. [Managed identities](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/overview) in Azure eliminate the need for developers to handle these credentials manually. MSALJS supports acquiring tokens through the managed identity service when used with applications running inside Azure infrastructure, such as:
+
```

Comment:
Pls add a note here that clarifies that only the Node version of MSALJS supports confidential client and managed identity. Browser-based MSAL JS does not offer confidential flows because there is no confidentiality between the browser and the token issuer and does not offer managed identity because the browser is not hosted on Azure.